e2e/specs/translation-builder-workflows.spec.ts

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable testing-library/prefer-screen-queries */
+import { test } from '../core';
+import { expect } from '@playwright/test';
+import { deleteForm, createForm, createValueReference, addFormResources } from '../commands/form-operations';
+import { FormBuilderPage } from '../pages';
+
+let formUuid = '';
+
+const formDetails = {
+  name: 'Covid-19 Screening',
+  description: 'A test form for recording COVID-19 screening information',
+  version: '1.0',
+  published: true,
+};
+
+test.describe('Translation Builder Workflows', () => {
+  test.beforeEach(async ({ api }) => {
+    const form = await createForm(api, false, formDetails);
+    formUuid = form.uuid;
+
+    const valueReference = await createValueReference(api);
+    await addFormResources(api, valueReference, formUuid);
+  });
+
+  test.afterEach(async ({ api }) => {
+    if (formUuid) {
+      await deleteForm(api, formUuid);
+    }
+  });
+
+  test('Workflow 1: Basic Translation Builder Display', async ({ page }) => {
```

Comment:
Can you check the other test on how we describe the test steps? 

https://github.com/openmrs/openmrs-esm-form-builder/blob/main/e2e/specs/delete-existing-form.spec.ts

ex: 

When I visit the form builder

And I search for the form and open it for editing

And I open the Translation builder tab

Then I should see the translation builder components (Download/Upload buttons, Translation filter tabs: All/Translated/Untranslated, and the search input)